Makefile

```diff
@@ -385,6 +395,14 @@ check-proto-diff:
 	git diff --exit-code ./pkg/proto/sentry/v1/sentry.pb.go # check no changes
 
 
+################################################################################
+# Target: compile-build-tools                                                              #
+################################################################################
+compile-build-tools:
+ifeq (,$(wildcard $(BUILD_TOOLS)))
+	cd build-tools; CGO_ENABLED=$(CGO) GOOS=$(TARGET_OS_LOCAL) GOARCH=$(TARGET_ARCH_LOCAL) go build -o $(BUILD_TOOLS_BIN) .
```

Comment:
Please, rename this folder to `.build-tools` to keep it separate from the main code.

Reply:
Done